grpc/src/main/java/com/linecorp/armeria/internal/client/grpc/ArmeriaClientCall.java

```diff
@@ -248,9 +248,10 @@ public void start(Listener<O> responseListener, Metadata metadata) {
 
         final BiFunction<ClientRequestContext, Throwable, HttpResponse> errorResponseFactory =
                 (unused, cause) -> HttpResponse.ofFailure(
-                        GrpcStatus.fromThrowable(exceptionHandler, ctx, cause, metadata)
-                                  .withDescription(cause.getMessage())
-                                  .asRuntimeException());
+                        DefaultGrpcExceptionHandlerFunction.fromThrowable(exceptionHandler, ctx, cause,
+                                                                          metadata)
+                                                           .withDescription(cause.getMessage())
+                                                           .asRuntimeException());
```

Comment:
Should we set `DefaultGrpcExceptionHandlerFunction` as a fallback exception handler for `exceptionHandler` when it is set in the `GrpcClientBuilder` like we do in `ServerBuilder`?
https://github.com/line/armeria/blob/ad3519fcc7ad1d0aa47ce15bd99a29ddced99f4a/core/src/main/java/com/linecorp/armeria/server/ServerBuilder.java#L2166-L2172
The rest of code should not access to `DefaultGrpcExceptionHandlerFunction` and use the composed one.

Reply:
Okay, I'll modify to use `DefaultGrpcExceptionHandlerFunction` as fallback exception handler.

Reply:
Global comments) If `DefaultGrpcExceptionHandlerFunction` is set as the fallback, should we remove `DefaultGrpcExceptionHandlerFunction` here and others?
```suggestion
                        exceptionHandler.apply(ctx, cause, metadata)
                                        .withDescription(cause.getMessage())
                                        .asRuntimeException());
```

Reply:
Applied feedback. Thanks